#
#                              /\
#                            /-  \
#                          /-     \
#                        /-        \
#              (L4)    /-           \
#                    /-  \           \
#                  /-     \           \
#                 /        \           \ (L2)
#               /-          \           \
#           M3/-             \        M1 \
#           /-                \           \
#         /-                   \           \
#       /-                   M2 \           \
#     /-                         \           \
#+------+                         \           \
#| MTool|                          \           \
#+------+                           \         q1-  (L0)
#                                    \          -------
#                                     \    (L1)      q0
#                                    q2              |
#                                                    |
#                                                    |
#                                                    |
#
EMU_MELB:
  dqMax: 360 # Max joint speed (deg.s-1) Set to 1 rot/s
  tauMax: 42 # Max joint torque (Nm) Set to max motor torque w/ 1:22 reduction (yes, this is the actual value!)

  linkLengths: [0.056, 0.135, 0.5, 0.46]   # Link lengths used for kinematic models (in m), excluding tool
  linkMasses: [0, 0.450, 0.400, 0.100, .0] # Link masses used for gravity compensation (in kg), excluding tool

  qLimits: [-45, 45, -15, 70, 0, 95] # Joints limits (in deg) {q1_min, q1_max, q2_min, q2_max, q3_min, q3_max}
  qCalibration: [-38, 70, 95] # Calibration configuration (in deg): posture in which the robot is when using the calibration procedure

  frictionVis: [0.2, 0.2, 0.2]  # Joint viscous friction compensation coefficients
  frictionCoul: [0.5, 0.5, 0.5] # Joint Coulomb (static) friction compensation coefficients

  #End-effector description
  #tool:
  #  name: "Handle"
  #  length: 0.14 #Tool length from attachment
  #  mass:   0.8  #Tool mass in kg